src/OperationalInsights/OperationalInsights/Az.OperationalInsights.psd1

```diff
@@ -73,68 +73,69 @@ NestedModules = @('Microsoft.Azure.PowerShell.Cmdlets.OperationalInsights.dll')
+               'Disable-AzOperationalInsightsLinuxSyslogCollection',
+               'New-AzOperationalInsightsWindowsEventDataSource',
+               'Get-AzOperationalInsightsSavedSearch',
+               'Get-AzOperationalInsightsSchema',
+               'New-AzOperationalInsightsComputerGroup',
+               'New-AzOperationalInsightsSavedSearch',
+               'Set-AzOperationalInsightsSavedSearch',
+               'Remove-AzOperationalInsightsSavedSearch',
+               'Get-AzOperationalInsightsDataSource',
+               'Remove-AzOperationalInsightsDataSource',
+               'Set-AzOperationalInsightsDataSource',
+               'Get-AzOperationalInsightsStorageInsight',
+               'Set-AzOperationalInsightsStorageInsight',
+               'New-AzOperationalInsightsStorageInsight',
+               'Remove-AzOperationalInsightsStorageInsight',
+               'Set-AzOperationalInsightsIntelligencePack',
+               'Get-AzOperationalInsightsIntelligencePack',
+               'Get-AzOperationalInsightsWorkspaceManagementGroup',
+               'Get-AzOperationalInsightsWorkspaceUsage',
+               'Get-AzOperationalInsightsWorkspaceSharedKey',
+               'Get-AzOperationalInsightsWorkspace',
+               'New-AzOperationalInsightsWorkspace',
+               'Remove-AzOperationalInsightsWorkspace',
+               'Set-AzOperationalInsightsWorkspace',
+               'Invoke-AzOperationalInsightsQuery',
+               'New-AzOperationalInsightsLinkedStorageAccount',
+               'Get-AzOperationalInsightsLinkedStorageAccount',
+               'Set-AzOperationalInsightsLinkedStorageAccount',
+               'Remove-AzOperationalInsightsLinkedStorageAccount',
+               'New-AzOperationalInsightsCluster',
+               'Get-AzOperationalInsightsCluster',
+               'Update-AzOperationalInsightsCluster',
+               'Remove-AzOperationalInsightsCluster',
+               'Get-AzOperationalInsightsLinkedService',
+               'Set-AzOperationalInsightsLinkedService',
+               'Remove-AzOperationalInsightsLinkedService',
+               'Get-AzOperationalInsightsDeletedWorkspace',
+               'Restore-AzOperationalInsightsWorkspace',
+               'Get-AzOperationalInsightsTable',
+               'Get-AzOperationalInsightsOperation',
+               'Get-AzOperationalInsightsDataExport',
+               'New-AzOperationalInsightsDataExport',
+               'Remove-AzOperationalInsightsDataExport',
+               'Update-AzOperationalInsightsDataExport',
+               'Get-AzOperationalInsightsOperationStatus',
+               'Update-AzOperationalInsightsWorkspaceSharedKey',
+               'New-AzOperationalInsightsPurgeWorkspace',
+               'Get-AzOperationalInsightsPurgeWorkspaceStatus',
+               'Get-AzOperationalInsightsAvailableServiceTier',
+               'Set-AzOperationalInsightsTable'
```

Comment:
I'm confused why we add 'Set-AzOperationalInsightsTable' in handcrafted cmdlet? I remember we replace it with generated cmdlet, right?